spec/lib/feature_management_spec.rb

```diff
@@ -181,49 +181,26 @@
     end
   end
 
-  describe '.no_pii_mode?' do
-    let(:proofing_vendor) { :mock }
-    let(:enable_identity_verification) { false }
-
-    before do
-      allow_any_instance_of(Figaro.env).to receive(:profile_proofing_vendor).
-        and_return(proofing_vendor)
-      allow(Figaro.env).to receive(:enable_identity_verification).
-        and_return(enable_identity_verification.to_json)
-    end
-
-    subject(:no_pii_mode?) { FeatureManagement.no_pii_mode? }
-
-    context 'with mock ID-proofing vendors' do
-      let(:proofing_vendor) { :mock }
-
-      context 'with identity verification enabled' do
-        let(:enable_identity_verification) { true }
-
-        it { expect(no_pii_mode?).to eq(true) }
-      end
-
-      context 'with identity verification disabled' do
-        let(:enable_identity_verification) { false }
+  describe '.fake_banner_mode?' do
+    subject(:fake_banner_mode?) { FeatureManagement.fake_banner_mode? }
 
-        it { expect(no_pii_mode?).to eq(false) }
+    context 'fake banner mode in production' do
+      it 'returns true' do
```

Comment:
I think you meant `it 'returns false'`? Also, what do you think of a more descriptive test title, such as:
```ruby
context 'when on secure.login.gov' do
  it 'does not display the fake banner' do
```

Reply:
DONE